tests/test_utils.py

```diff
@@ -201,6 +217,31 @@ def test_deserialize(self):
         self.assertTrue(result.is_a(city.Citizen))
         self.assertEqual(result.name, "Peter")
         self.assertEqual(result.age, 23)
+        result = deserialize([CUDS_DICT])
+        self.assertEqual(len(result), 1)
+        self.assertTrue(result[0].is_a(city.Citizen))
+        self.assertEqual(result[0].name, "Peter")
+        self.assertEqual(result[0].age, 23)
+        self.assertEqual(CUDS_LIST, serialize(deserialize(CUDS_LIST)[0],
+                         json_dumps=False))
+
+    def test_serialize(self):
+        c = branch(
+            city.City(name="Freiburg", uid=1),
+            branch(
+                city.Neighborhood(name="Littenweiler", uid=2),
+                city.Street(name="Schwarzwaldstraße", uid=3)
+            )
+        )
+        self.maxDiff = None
```

Comment:
What does this do?

Reply:
The self.maxDiff? In case there is an error in the unittest, it will print the entire diff and not only one line